NGCHM/WebContent/javascript/custom/custom.js

```diff
@@ -239,8 +239,8 @@ function linkoutHelp () {
         name: "CIViC",
         description: "Adds linkouts to the CIViC Mutation Database",
         version: "0.1.0",
-        site: "https://civic.genome.wustl.edu/home",
-        logo: "https://civic.genome.wustl.edu/assets/images/CIViC_logo.png",
+        site: "https://civic.genome.wustl.edu/welcome",
```

Comment:
@marohrdanz How about just https://civicdb.org ?  which appears to get to the same place.

Reply:
That seems like a good choice, and I made that change.